include/bluetooth/services/fast_pair.h

```diff
@@ -20,7 +20,14 @@
 extern "C" {
 #endif
 
-/** @brief Fast Pair advertising mode. Used to generate advertising packet. */
+/** Value that denotes unknown battery level (see @ref bt_fast_pair_battery_value). */
+#define BT_FAST_PAIR_UNKNOWN_BATTERY_LEVEL	0x7f
```

Comment:
Please use the structure prefix to nudge the user where this definition can be used

```suggestion
#define BT_FAST_PAIR_BATTERY_VALUE_LEVEL_UNKNOWN	(0x7f)
```

or

```suggestion
#define BT_FAST_PAIR_BATTERY_VALUE_UNKNOWN	(0x7f)
```